Generating New Applications.md

```diff
@@ -1,23 +1,26 @@
-## PreCreation Checkup (Team Lead)
+## PreCreation checklist (Devops)
 
 Before a developer creates a new application, a couple of 3rd party services need to be setup.
 
-* **Bitbucket repository** - Create a repository for the project with template: 'web_{app_name}'
+* **Bitbucket repository** - Create a repository for the project with template: [web-{app_name}](https://github.com/infinum/internal-guides/blob/master/How%20to%20name%20your%20repository.md)
```

Comment:
This is a public chapter and you're linking to a private repository. I would delete the reference to it since I hope we will finally delete it some day.